### PR TITLE
Add config for new tier & restrict access for now

### PR DESCRIPTION
**Pull request by davidrapson**
This is work from @wpf500 to enable a new tier. Just opening it up as a PR so we have some more visibility on it.

### CODE REVIEW

File: frontend/conf/application.conf [hunk 1]
```diff
@@ -87,6 +89,10 @@ touchpoint.backend {
 
                 friend = ""
                 staff = ""
+                supporter {
+                    monthly = ""
+                    annual = ""
+                }
```
Comment (jennysivapalan) [inline frontend/conf/application.conf]: I've added the relevant properties in private configs for supporter. 
I assume the facebooke and google stuff can be added in here.

**Comment by jennysivapalan** [comment]
I've just updated this branch to keep with master. @jamesoram while the copy isn't ready would you be able to test the end to end of this and make sure we get the tier set up in salesforce etc?

**Comment by jamesoram** [comment]
I can take a look, but I don't have access to SF anymore.

**Comment by jennysivapalan** [comment]
@rtyley @davidrapson I've added a commit which would be the paths to cut off Supporter access for now (register and upgrade). https://github.com/guardian/membership-frontend/commit/3a917bbfeedbf55c5e6566dfc4b3ecf0079237a7?w=1

@jamesoram If you haven't spoken to Touchpoint yet let me know and we can try and sort it this afternoon.

**Comment by jamesoram** [comment]
I will have to work with Jason on this.

**Comment by rtyley** [comment]
I've rebased this PR against master, and squashed the work down into fewer commits - it was quite hard to see that not much rocket science was going on! The exciting bit was adding the security requested by Jason Burr - [`secureHiddenTiers`](https://github.com/guardian/membership-frontend/blob/952804c/frontend/app/controllers/Joiner.scala#L28) will restrict access to membership.dev@guardian.co.uk & touchpoint@guardian.co.uk for now.

This is ready for merge as far as I'm concerned, once in, authorised users will be able to see
https://membership.theguardian.com/join/supporter/enter-details - which till needs several fixes to the text around pricing.

**Comment by jamesoram** [comment]
:+1:

**Comment by rtyley** [comment]
This is on Production now: https://membership.theguardian.com/join/supporter/enter-details - **but the thank you page is failing** : https://app.getsentry.com/the-guardian/membership/group/56964217/ - looking into this now. I've already re-verified that I can join as a Friend (real user) and Partner (test-user) on Production.

Note also that we need to tweak the '(save £0 a year)' message:

image

**Comment by rtyley** [comment]
Verified that in DEV, I can join as a Supporter against the DEV Touchpoint backend:

image

**Comment by rtyley** [comment]
Verified that in DEV (my local machine), I can join as a Supporter against the UAT Touchpoint backend:

image


https://apisandbox.zuora.com/apps/CustomerAccount.do?method=view&id=2c92c0f84be36372014beeccb9da1169 <-- UAT Zuora

**Comment by rtyley** [comment]
Fix for this thank-you page crash is going in with #287

**Comment by rtyley** [comment]
With #287, the functionality is finally successful on PROD: image

**Pull request merged**